prometheus_metrics/prometheus_metrics.py

```diff
@@ -17,6 +19,8 @@
 }
 
 
-def generate_latest_metrics():
-    """Generate Latest."""
-    return generate_latest()
+def generate_aggregated_metrics():
```

Comment:
This change finally justifies the definition of `generate_*_metrics` function. Before, it wasn't good, now it's much better!

Reply:
Agree - now it has a real purpose.